src/config.rs

```diff
@@ -679,6 +680,10 @@ impl DefaultCfConfig {
             prop_size_index_distance: self.prop_size_index_distance,
             prop_keys_index_distance: self.prop_keys_index_distance,
         };
+        cf_opts.add_table_properties_collector_factory(
+            "tikv.mvcc-properties-collector",
```

Comment:
Should we use another name being different with `MvccPropertiesCollectorFactory` ?